File: pe020/pe020.go
/*
Project Euler, Problem 20:

Find the sum of the digits in the number 100!

https://projecteuler.net/problem=20
*/

package main

import (
	"fmt"
	"math/big"
	"strconv"
)

func main() {
	var sum int
	var bigint big.Int
	for _, s := range bigint.MulRange(1, 100).String() {
		d, _ := strconv.Atoi(string(s))
		sum += d
	}
	fmt.Println(sum)
}

File: pe020/pe020.py
#!/usr/bin/python

"""
Project Euler, Problem 20:

Find the sum of the digits in the number 100!

https://projecteuler.net/problem=20
"""

def fact(n):
    if n == 1:
        return 1
    return n * fact(n - 1)

print(sum([ int(s) for s in str(fact(100)) ]))
